plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/N4JSScopeProvider.xtend

```diff
@@ -252,7 +255,7 @@ class N4JSScopeProvider extends AbstractScopeProvider implements IDelegatingScop
 	}
 
 	override getScopeForContentAssist(EObject context, EReference reference) {
-
+		suppressAccessKindOnMemberScopeRequest = true;
```

Comment:
It's hacky to pass through this argument along the call hierarchy by way of a field. But I get why it's difficult to avoid and we already have a case like this (cf. `suppressCrossFileResolutionOfIdentifierRef`).

But at least you have to make sure that setting this value does not affect later invocations of `#getScopeForContext()`.